test/nn/test_pooling.py

```diff
@@ -1421,6 +1421,18 @@ def test_adaptive_pool_invalid(self, device):
                 t, output_size = inp
                 m(output_size)(t)
 
+    @slowTest
+    def test_adaptive_pool_odd_size(self, device):
+        # See https://github.com/pytorch/pytorch/issues/81409
+        Ih, Iw, Oh, Ow = 5873, 3693, 3527, 2219
```

Comment:
These are specifically chosen numbers that are causing an old version to segfault in #81409